tests/README.md

```diff
@@ -244,3 +249,8 @@ pytest tests/smoke/test_notebooks_python.py::test_sar_single_node_smoke
 ```
```

Comment:
The bold of `**Note that the next instructions execute the tests from the root folder.**` is confused with the title, now that we have more levels. Maybe put italic or `**NOTE:** the next instructions execute the tests from the root folder` ?